bin/helpers.sh

```diff
@@ -2,14 +2,31 @@
 
 # Arbitrary download function that uses wget or curl depending on what's available.
 download() {
-    if which curl &> /dev/null; then  
-        curl -s "$1" > "$2";  
-    elif which wget &> /dev/null; then  
-        wget -nv -O "$2" "$1"  
-    else  
-        echo "Missing curl or wget" >&2  
-        exit 1  
-    fi  
+	if which curl &> /dev/null; then  
+		curl -s "$1" > "$2";  
+	elif which wget &> /dev/null; then  
+		wget -nv -O "$2" "$1"  
+	else  
+		echo "Missing curl or wget" >&2  
+		exit 1  
+	fi  
+}
+
+# Function to ensure svn is installed
+ensure_svn_installed() {
+	if ! which svn &> /dev/null; then
+		echo "svn is not installed. Installing..."
+		if which apt-get &> /dev/null; then
+			sudo apt-get update && sudo apt-get install -y subversion
```

Comment:
```suggestion
			sudo apt-get update
			sudo apt-get install -y subversion
```
it's a shell script. no reason to make this one line.